<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editTeams</class>
 <widget class="QDialog" name="editTeams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="submitEditTeam">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QComboBox" name="editTeam">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ADD</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>REMOVE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Team</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editTeamName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Team Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitTeam">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Team Mgr.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="teamManager">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Identification</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="teamID">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
